src/pvq_encoder.c

```diff
@@ -114,7 +115,16 @@ static double pvq_search_rdo_double(const od_val16 *xcoeff, int n, int k,
   if (prev_k > 0 && prev_k <= k) {
     /* We reuse pulses from a previous search so we don't have to search them
        again. */
+    double l1_norm;
+    double l1_inv;
+    l1_norm = 0;
+    for (j = 0; j < n; j++) l1_norm += x[j];
+    l1_inv = 1./OD_MAXF(l1_norm, 1e-100);
```

Comment:
Is the `OD_MAXF` to avoid divisions by zero? Can this happen since we are in a block where k > 0 ?